cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -95,6 +95,9 @@ type engineExistenceGuarantor struct {
 
 	// Destinations the logs aggregator will deliver to
 	sinks logs_aggregator.Sinks
+
+	// If set to true, engine will not store logs in a persistent volume
+	shouldTurnOffPersistentVolumeLogsCollection bool
```

Comment:
I find that negatives make things harder to reason about, and increase the risk of bugs that someone will use the bool in the opposite way intended. Suggest instead, `shouldEnablePersistentVolumeLogsCollection`

Reply:
ye i agree with this - i went with the negative that it was clear that the default is that `PersistentVolumeLogsCollection` is turned of - if it's not the negative then `shouldEnablePersistentVolumeLogsCollection` seems like it's not the default

Reply:
i think the right move is to go with the positive + set default to true + comment why thats the default , warn against changing